kratos/geometries/geometry.h

```diff
@@ -592,12 +592,6 @@ class Geometry
     ///@name Id
     ///@{
 
-    /// Id of this Geometry
-    IndexType Id()
```

Comment:
is this intended?

Reply:
I think the non const version of the Id access is not needed because it shouldn't be changed from outside anyways. Or am I wrong here?

Reply:
@pooyan-dadvand and I decided to drop the non-const access to the Id.

Reply:
It returns a copy of Id and being non const is not useful. So IMO only the const version is enough.